tools/functional/conftest.py

```diff
@@ -17,27 +22,7 @@ def random_id():
 
 @pytest.fixture(scope="session")
 def CONFIG():
-    return "/tmp/python-gitlab.cfg"
-
-
-@pytest.fixture
-def gitlab_cli(script_runner, CONFIG):
-    """Wrapper fixture to help make test cases less verbose."""
-
-    def _gitlab_cli(subcommands):
-        """
-        Return a script_runner.run method that takes a default gitlab
-        command, and subcommands passed as arguments inside test cases.
-        """
-        command = ["gitlab", "--config-file", CONFIG]
-
-        for subcommand in subcommands:
-            # ensure we get strings (e.g from IDs)
-            command.append(str(subcommand))
-
-        return script_runner.run(*command)
-
-    return _gitlab_cli
+    return os.path.join(TEMP_DIR, "python-gitlab.cfg")
```

Comment:
Suggestion: Might as well use pathlib here, as we're Python 3.6+ 😄 

```python
from pathlib import Path
```

```suggestion
    return Path(TEMP_DIR) / "python-gitlab.cfg"
```

Reply:
Thank you! Ah yes, definitely don't want to bring back python2 ghosts 😁 fixed.

Reply:
Both are valid approaches still I guess, but Pathlib is kind of nicer and we can use it now 😄